Fruitties/iosApp/iosApp/ui/CartView.swift

```diff
@@ -19,13 +19,24 @@ import SwiftUI
 import shared
 
 struct CartView : View {
-    let cartViewModel: CartViewModel
+    /// Injects the `IOSViewModelStoreOwner` from the environment, which manages the lifecycle of `ViewModel` instances.
+    @EnvironmentObject var viewModelStoreOwner: IOSViewModelStoreOwner
```

Comment:
When is the `viewModelStoreOwner` cleared?

Reply:
We have the `ViewModelStoreOwnerProvider` which calls clear with `onDisappear` of the `content` View.